cumulusci/tasks/github/release.py

```diff
@@ -35,6 +35,9 @@ class CreateRelease(BaseGithubTask):
         "resolution_strategy": {
             "description": "The name of a sequence of resolution_strategy (from project__dependency_resolutions) to apply to dynamic dependencies. Defaults to 'production'."
         },
+        "package_type": {
```

Comment:
What do you think about making this required? If it's not populated, going forward, we will assume 1GP (those releases being created by an older version of CumulusCI), so any 2GP release _must_ declare itself as a 2GP release.

Reply:
I was trying to think through the implications of making it required; I suppose I didn't want to break anyone who is using this in a CI system and figured that since we will adhere to using our standard library release flows, that these would always be populated for us (so long as we're hardcoding that value in the flows). Making it required, would come with the benefit you noted though. I'll go ahead and switch it.

Reply:
We should probably call that out as a Critical Change. If someone has built their own flows rather than using ours, they'll break, although they will fail safe.

Reply:
I went ahead and called it out in the Critical Changes section.